data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"Great! So, Anna, what are we doing next?"`
+			`	"Getting my book published, of course. We need to find a publishing company!"`
+			`	"We should go for the Kimek", you say. "Their population is 200 billion. For comparison, the human population is only 100 billion."`
+			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
+			choice
+				`	"We'll find a company. There has to be one out there."`
+			`	"The faster we can get the book published, the better. I'm off to the spaceport now. Meet me in the spaceport once you've found a good company."`
+
+mission "Fangvin: Struggling Author 4"
+	name "Kimek Publishing Company"
+	description "You have found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	to offer
+		has "Fangvin: Struggling Author 3: done"
+	on offer
+		conversation
+			`The spaceport of Ki Patek Ka is vast. You decide to search for Anna, who has chosen to settle in the cafeteria, with her laptop, She is typing away at the keyboard, presumably writing a new novel.`
+			`	"<first>! I found a publishing company who's willing to publish my book! I sent them a digital copy of my book and they have instructed me to visit their office on <destination>!", exclaims Anna, once again in an enthusiastic tone.`
+			choice
+				`	"What's the publishing company called?"`
+			`	"They're called 'Kimek Reader Publishing'.`
+			choice
+				`	"Great!"`
+				`	"Let's go!"`
+			`	Anna closes her laptop, places her things in her bag, and begins walking to your ship. You and Anna enter the ship, as she settles into her bunk. It's time to go to <destination>, where the main offices of 'Kimek Reader Publishing' are located.`
+				accept
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive', her novel about humans stealing jump drives from the Quarg. She hastily opens the hatch as soon as it unlocks.`
+			choice
+				`	"Wait for me!"`
+			`	Anna hears your scream and stops. "We're here, Anna. Now, we need to find their offices.", you say.`
+			`	"That won't be necessary. In the message they sent me, they included precise instructions on how to reach their offices."`
+			`	You begin following Anna to the publisher's offices. You walk through endless rows of shops and streets, take a train to a large white building, and eventually find what looks like the Kimek equivalent of an elevator.`
+			`	You and Anna enter the elevator, as Anna presses a button that closes the door and takes you up. When the door opens, you see a Kimek in front of you, a necklace around it with ornate jewelry and a translation device.`
+			`	"Greetings, humans. Follow me, you must, so publish your book, we can.", says the Kimek through its translation box.`
```

Comment:
I think they'd first talk about the whole thing a lot more rather than just getting straight to publishing. There's a timer of many months, so that does seem to be supported by the timescale, so just tweak this dialog so that it's not implying they'll go straight to talking about publishing.

Reply:
```suggestion
			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," says the Kimek through its translation box.`
```